app/handlers/karma.py

```diff
@@ -46,6 +52,11 @@ async def get_top(
     chat_repo: ChatRepo,
     user_repo: UserRepo,
 ):
+    parts = message.text.split(maxsplit=1)
+    if len(parts) > 1:
+        return await message.reply(
+            "Просмотр топа другого чата возможен только в личных сообщениях с ботом"
+        )
```

Comment:
let's remove this error message after 60 second. and message with command from user too